pyproject.toml

```diff
@@ -81,6 +81,7 @@ dev-pinned = [
     "coverage==7.2.5",
     "cssbeautifier==1.14.7",
     "dill==0.3.6",
+    "django-livereload-server",
```

Comment:
Only adding it to `dev-pinned` without adding it to `dev` will remove it on the next package update.
The idea is that the `dev-pinned` extra is never updated manually, but just via the update script.